cli/src/index.js

```diff
@@ -117,6 +127,21 @@ yargsInstance
     run
   )
 
+  .command(
+    'mock <inputType>',
+    'Create a mocked input for test',
+    (yargs) =>
+      yargs.positional('inputType', {
+        describe: 'Type of input to mock',
+        choices: ['protectedData'],
+      }),
+    ({ inputType, ...argv }) => {
+      if (inputType === 'protectedData') {
+        return mockProtectedData(argv);
+      }
+    }
+  )
+
```

Comment:
Currently, the UI allows users to input any mock data, but the only valid input is `protectedData`. It's a little bit confusing, may be we can consider one of the following approaches:
- Use a select prompt type to guide users towards the correct input.
- Replace the command with a more specific one, like `iapp mockProtectedData`.

These changes would ensure that users have clear guidance on what inputs are valid

Reply:
I partially agreed, I did it because I like sub-commands style `iapp mock protectedData` looks better than `iapp mockProtectedData`
I `iapp` command semantic second arg (`init`, `test`, `deploy`, `run`) is an action verb, precision can be added with third argument...

An alternative I consider to be fine is to set a default value for [inputType="protectedData"] `iapp mock protectedData` would be aliased by `iapp mock`

Reply:
Yes, I understand that the command should be a verb, but inputType might make the user think that any input can be entered (personal point of view). Maybe we could ask @cedric25 what he thinks.